app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
__Same here__
- [ ] [OPTIONAL] Instead of calling count on records (image 1000 records, it will slow the application), we can simply use the ...increment... Ruby method. Something like this: 
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```
__Since you are using an after_save callback function, this function should be a private function.__